src/garage/torch/modules/gaussian_mlp_module.py

```diff
@@ -0,0 +1,223 @@
+        hidden_w_init (callable): Initializer function for the weight
+            of intermediate dense layer(s). The function should return a
+            torch.Tensor.
+        hidden_b_init (callable): Initializer function for the bias
+            of intermediate dense layer(s). The function should return a
+            torch.Tensor.
+        output_nonlinearity (callable): Activation function for output dense
+            layer. It should return a torch.Tensor. Set it to None to
+            maintain a linear activation.
+        output_w_init (callable): Initializer function for the weight
+            of output dense layer(s). The function should return a
+            torch.Tensor.
+        output_b_init (callable): Initializer function for the bias
+            of output dense layer(s). The function should return a
+            torch.Tensor.
+        learn_std (bool): Is std trainable.
+        init_std (float): Initial value for std (normal value).
+        adaptive_std (bool): Is std a neural network. If False, it will be a
+            parameter.
+        std_share_network (bool): Boolean for whether mean and std share
+            the same network.
+        std_hidden_sizes (list[int]): Output dimension of dense layer(s) for
+            the MLP for std. For example, (32, 32) means the MLP consists
+            of two hidden layers, each with 32 hidden units.
+        min_std (float): If not None, the std is at least the value of min_std,
+            to avoid numerical issues (normal value).
+        max_std (float): If not None, the std is at most the value of max_std,
+            to avoid numerical issues (normal value).
+        std_hidden_nonlinearity: Nonlinearity for each hidden layer in
+            the std network.
+        std_output_nonlinearity (callable): Activation function for output
+            dense layer in the std network. It should return a torch.Tensor. Set
+            it to None to maintain a linear activation.
+        std_output_w_init (callable): Initializer function for the weight
+            of output dense layer(s) in the std network.
+        std_parametrization (str): How the std should be parametrized. There
+            are two options:
+            - exp: the logarithm of the std will be stored, and applied a
+               exponential transformation
+            - softplus: the std will be computed as log(1+exp(x))
+        layer_normalization (bool): Bool for using layer normalization or not.
+    """
+
+    def __init__(self,
+                 input_dim,
+                 output_dim,
+                 hidden_sizes=(32, 32),
+                 hidden_nonlinearity=torch.tanh,
+                 hidden_w_init=nn.init.xavier_uniform_,
+                 hidden_b_init=nn.init.zeros_,
+                 output_nonlinearity=None,
+                 output_w_init=nn.init.xavier_uniform_,
+                 output_b_init=nn.init.zeros_,
+                 learn_std=True,
```

Comment:
This argument is ignored. As far as I can tell, it should determine if `_init_std` is a `torch.nn.Parameter` or just a `torch.tensor`?

Reply:
hmm I assume init_std is float type. like other min_std and max_std.
Do I need to get init_std as torch object(Parameter or tensor)?

Reply:
Got it!

Reply:
Make default value of init_std as torch.nn.Parameter because in TF version, default value if  learn_std is True

Reply:
I would just wrap `init_std` when you assign it to `self._init_std_param`, but otherwise yeah.

Reply:
Is it ok to use requires_grad?
like
`self._init_std_param = torch.tensor(init_std, dtype=torch.float, requires_grad=self._learn_std)`

Reply:
That might break other things in the future, since it e.g. won't be listed in the `state_dict` just use:
```python
self._init_std_param = nn.Parameter(init_std)
```

Reply:
ok, fixed